pkg/kubelet/allocation/allocation_manager.go

```diff
@@ -214,6 +232,37 @@ func (m *manager) Run(ctx context.Context) {
 	}()
 }
 
+// Gernerate pod resize completed event
+func (m *manager) emitPodResizeCompletionEvent(resizedPod *v1.Pod) string {
+	podResizeSource := &podResourceSummary{
+		InitContainers: make([]containerAllocation, 0),
+		Containers:     make([]containerAllocation, 0),
+	}
+
+	podutil.VisitContainers(&resizedPod.Spec, podutil.InitContainers|podutil.Containers,
+		func(allocatedContainer *v1.Container, containerType podutil.ContainerType) bool {
```

Comment:
(optional) `podutil.ContainerIter` would be a little cleaner here


```suggestion
	for allocatedContainer, containerType := range podutil.ContainerIter(&resizedPod.Spec, podutil.InitContainers|podutil.Containers) {
```